boards/openhwgroup/cv32a6_genesys_2/cv32a6_genesys_2.dts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2024 CISPA Helmholtz Center for Information Security gGmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/dts-v1/;
+
+#include <openhwgroup/cv32a6.dtsi>
+
+/ {
+	model = "Openhardwaregroup CV32A6 on Genesys II";
```

Comment:
Looks like we're still using the old name here

Reply:
Oops, I missed them. I have pushed a fix.